### PR TITLE
correct roads aggregation alignments df7 10

### PR DESCRIPTION
**Pull request by annat2022**
SVC-1696

### CODE REVIEW

File: Aggregation/MajorRoads/DF7_10_Roads_Aggregations.halex.alignment.xml [hunk 1]
```diff
@@ -373,7 +373,7 @@ _snippets.end_Checks {&#13;
 	//To Do: END_RT44 Mismatching of roadIdIdentifier for actionPlanId_identifier between the tables: NAP_MajorRoad and NAP_RoadReductionMeasure -&gt; für UBA-DE theoretisch nicht relevant, da keine RoadIDs?&#13;
 	&#13;
 	nandCheck('END_TO27', SHEET, ['existingMeasure', 'plannedMeasureDetail_plannedMeasure'], 'The fields existingMeasure and plannedMeasureDetail_plannedMeasure must not both contain the value noMeasure', isEqualTo('noMeasure'))&#13;
-	conditionalCheck('END_TO26', SHEET, 'plannedMeasureDetail_allMeasuresInCost', 'plannedMeasureDetail_measuresInCost', String message, isEqualTo('No'), isNotEqualTo('noMeasure'))&#13;
+	conditionalCheck('END_TO26', SHEET, 'plannedMeasureDetail_allMeasuresInCost', 'plannedMeasureDetail_measuresInCost', 'The field measuresInCost must not contain the value noMeasure when all allMeasureInCost is No for that action plan', isEqualTo('No'), isNotEqualTo('noMeasure'))&#13;
```
Comment (JohannaOtt) [inline Aggregation/MajorRoads/DF7_10_Roads_Aggregations.halex.alignment.xml]: Is there an "all" too much between "when" and "allMeasureInCost"?

**Review by JohannaOtt — COMMENTED** [review]
@annat2022 Any need to review that content wise? In that case I would need to get into END first.

File: Aggregation/MajorRoads/DF7_10_Roads_Aggregations.halex.alignment.xml (hunk 1 above)
Comment (annat2022) [reply Aggregation/MajorRoads/DF7_10_Roads_Aggregations.halex.alignment.xml]: @JohannaOtt Thanks for the hint, I corrected this. (I also corrected the attribute name 'allMeasureInCost' to 'allMeasuresInCost'.)

**Comment by annat2022** [comment]
> Any need to review that content wise? In that case I would need to get into END first.

@JohannaOtt I'd say not in this case. But maybe @florianesser you could confirm if removing the used but not defined variable 'id' in the 'generalCheck' function is correct or if I missed something.

File: Snippets/end.Checks.groovy [hunk 2]
```diff
@@ -366,7 +366,7 @@ def generalCheck(String codeInvalid, String sheet, List fields, String message,
 		def valid = validator.call(['value':values.join('+'), 'sheet':sheet, 'field':fields.join('+')])
 		
 		if (!valid) {
-			_log.error("${codeInvalid}: The values ${values} for the fields ${fields} ${message} [${sheet}]${id}")
+			_log.error("${codeInvalid}: The values ${values} for the fields ${fields} ${message} [${sheet}]")
```
Comment (florianesser) [inline Snippets/end.Checks.groovy:369]: @annat2022 `${id}` can be safely removed here. It was most likely an oversight by me when I copied [this line](https://github.com/wetransform/uba-end-transformations/blame/d4d4613ba50dab4a335e80f7016aec4758378e7c/Snippets/end.Checks.groovy#L402).

**Review by florianesser — APPROVED** [review]
LGTM

File: Snippets/end.Checks.groovy (hunk 2 above)
Comment (annat2022) [reply Snippets/end.Checks.groovy:369]: That's what I assumed, just wanted to make sure.

**Pull request merged**